examples/migrate-pvc/README.MD

```diff
@@ -0,0 +1,125 @@
+# Migrate a PVC from a kubernetes cluster to another one 
+
+We'll demonstrate how to migrate a PVC from a cluster (source cluster )
+to another one (destination cluster) with Kanister.
+
+This example was tested to migrate a PVC from an openshift 3.9 cluster to an openshift 4.7 cluster.
+It's for this reason we install kanister in version 0.61.0 because it's the last kanister version still 
+compatible with openshift 3.9. If you don't have this constraint you can install kanister in the latest
+version.
+
+## Requirement 
+
+The PVC must not be mounted by any pods during the backup or during the restore.
+
+## Prerequisite  
+
+
+### Install kanister on both cluster 
```

Comment:
instead of giving the command to install kanister, and kanctl can be just mention the prerequisite like we are doing for apps in `examples/stable`.

Reply:
ok